src/Microsoft.Health.Fhir.Shared.Core/Features/Search/SearchOptionsFactory.cs

```diff
@@ -90,6 +90,17 @@ public SearchOptions Create(string compartmentType, string compartmentId, string
                     // Query parameter with empty value is not supported.
                     unsupportedSearchParameters.Add(query);
                 }
+                else if (query.Item1 == KnownQueryParameterNames.Total)
```

Comment:
This should probably also be StringComparison.OrdinalIgnoreCase

Reply:
Thanks, fixed in 9df2fc7.